### PR TITLE
App: Update for new protocol

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: packages/cli/src/commands/mint.js [hunk 1]
```diff
@@ -5,18 +5,18 @@ const command = 'mint'
...
-  const court = await environment.getCourt()
-  const to = recipient || await court.environment.getSender()
+  const protocol = await environment.getProtocol()
+  const to = recipient || await protocol.environment.getSender()
 
   let token
-  if (symbol.toLowerCase() === 'anj') token = await court.anj()
-  if (symbol.toLowerCase() === 'fee') token = await court.feeToken()
+  if (symbol.toLowerCase() === 'ant') token = await protocol.token()
+  if (symbol.toLowerCase() === 'fee') token = await protocol.feeToken()
```
Comment (promaty) [inline packages/cli/src/commands/mint.js:19]: Why are fees paid using a different token?

Comment (promaty) [inline packages/cli/src/commands/mint.js:19]: Why are fees paid using a different token?

File: packages/shared/models/artifacts/DynamicArtifacts.js [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
+    if (!artifact) throw Error(`Could not find artifact for ${dependency} ${contractName}`)
+    return artifact
+  }
+
+  _findArtifact(paths) {
+    const path = paths.find(fs.existsSync)
+    return path ? require(path) : undefined
   }
 
   _getLocalBuildPath(contractName) {
```
Comment (promaty) [inline packages/shared/models/artifacts/DynamicArtifacts.js]:
```suggestion
  _getLocalBuildPaths(contractName) {
```

File: packages/shared/models/artifacts/DynamicArtifacts.js [hunk 3]
```diff
@@ -1,21 +1,31 @@
...
-    return path.resolve(process.cwd(), `./build/contracts/${contractName}.json`)
+    return BUILD_DIRS.map(dir => path.resolve(process.cwd(), `./${dir}/${contractName}.json`))
   }
 
   _getNodeModulesPath(dependency, contractName) {
```
Comment (promaty) [inline packages/shared/models/artifacts/DynamicArtifacts.js]:
```suggestion
  _getNodeModulesPaths(dependency, contractName) {
```

File: packages/shared/models/artifacts/DynamicArtifacts.js [hunk 4]
```diff
@@ -1,21 +1,31 @@
...
 const BaseArtifacts = require('./BaseArtifacts')
 
+const BUILD_DIRS = ['build/contracts', 'artifacts']
+
 class DynamicArtifacts extends BaseArtifacts {
   getContractSchema(contractName, dependency = undefined) {
-    const contractPath = dependency
+    const contractPaths = dependency
       ? this._getNodeModulesPath(dependency, contractName)
       : this._getLocalBuildPath(contractName)
```
Comment (promaty) [inline packages/shared/models/artifacts/DynamicArtifacts.js]:
```suggestion
      ? this._getNodeModulesPaths(dependency, contractName)
      : this._getLocalBuildPaths(contractName)
```

File: packages/shared/models/artifacts/StaticArtifacts.js [hunk 5]
```diff
@@ -4,14 +4,12 @@ const BUILDS = {
...
   },
-  '@aragon/minime': {
-    'MiniMeToken': require('@aragon/minime/build/contracts/MiniMeToken')
-  },
-  '@aragon/court': {
-    'AragonCourt': require('@aragon/court/build/contracts/AragonCourt'),
-    'DisputeManager': require('@aragon/court/build/contracts/DisputeManager'),
-    'JurorsRegistry': require('@aragon/court/build/contracts/JurorsRegistry'),
-    'CourtSubscriptions': require('@aragon/court/build/contracts/CourtSubscriptions'),
+  '@aragon/protocol-evm': {
```
Comment (promaty) [inline packages/shared/models/artifacts/StaticArtifacts.js:7]: why are you calling this `aragon/protocol-evm`? this suggests that you are returning a dedicated virtual machine or something

**Review by promaty — APPROVED** [review]

File: .env.sample [hunk 6]
```diff
@@ -37,12 +37,12 @@ WEB3_POLLING_INTERVAL=10000
 PRIVATE_KEY=0x93a05d24ed616a3b525087cf4c32fc21c4c1cbc7dba8d35e7f255385f4df192c
 
 # Web3 connection config
-NETWORK=rpc
+NETWORK=ganache
 RPC=http://localhost:8545
 
 # Court config
 CLIENT_URL=https://court.aragon.org/
```
Comment (promaty) [inline .env.sample:44]: What will be new client url?

File: packages/cli/src/commands/mint.js (hunk 1 above)
Comment (facuspagnuolo) [reply packages/cli/src/commands/mint.js:19]: these are not fees, simply tokens, the thing is that we are getting rid of ANJ in favor of ANT

File: .env.sample (hunk 6 above)
Comment (facuspagnuolo) [reply .env.sample:44]: `protocol.aragon.org`

File: packages/cli/src/commands/mint.js (hunk 1 above)
Comment (facuspagnuolo) [reply packages/cli/src/commands/mint.js:19]: these are not fees, simply tokens, the thing is that we are getting rid of ANJ in favor of ANT

File: packages/shared/models/artifacts/StaticArtifacts.js (hunk 5 above)
Comment (facuspagnuolo) [reply packages/shared/models/artifacts/StaticArtifacts.js:7]: simply identifying the package for those artifacts

**Pull request merged**